utils/vlog/yosys/json.py

```diff
@@ -42,7 +42,7 @@ def cells(self):
         type: str
         """
         clist = []
-        for cell, cdata in self.data["cells"].items():
+        for cell, cdata in sorted(self.data["cells"].items()):
             if cell.startswith("$"):
```

Comment:
Why do we skip cells which start with `$`?

Reply:
They are Yosys internal cells, most likely the result of RTL from a simulation model, and won't have a corresponding pb_type.